### PR TITLE
Update README.md documentation

### PR DESCRIPTION
**Pull request by fenrisl**
#38 

Update README.md documentation to use Environment object instead of Criticality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,17 +78,15 @@ os                      : @{key=windows_10; name=Windows 10; arch=; eol=2025-10-
 ```powershell
 PS> $client.remote_accesses()
 
+
```
Comment (hezanathos) [inline README.md:81]: ?

File: README.md [hunk 2]
```diff
@@ -94,28 +95,25 @@ node       : @{id=1; name=mynode; created_at=2018-09-12T17:16:02.000+02:00; upda
...
-PS > $ram_params = @{
-        type= "CbwRam::RemoteAccess::WinRm::WithNegotiate"
-        address= "test.com"
-        port= "5985"
-        login= "myLogin"
-        password= "myPassword"
-        node= "myNodeName"
+PS > $Info = @{
+    "address" = "12.34.56.78"
+    "type" = "CbwRam::RemoteAccess::Snmp"
```
Comment (hezanathos) [inline README.md:100]: Update the example is great. But please keep a winrm example.

File: README.md [hunk 3]
```diff
@@ -124,16 +120,14 @@ server     : @{id=18d2fc32acf9572830685df73b8fcf62; hostname=test.com; last_comm
...
 type       : CbwRam::RemoteAccess::WinRm::WithNegotiate
 address    : test.com
 port       : 5985
-is_valid   : False
-created_at : 2019-05-21T13:44:29.000+02:00
-updated_at : 2019-05-21T13:44:29.000+02:00
-server     : @{id=18d2fc32acf9572830685df73b8fcf62; hostname=test.com; last_communication=; reboot_required=; agent_version=; remote_ip=test.com; 
-             boot_at=; criticality=criticality_medium; updates_count=0; cve_announcements_count=0; category=server}
-
+is_valid   :
```
Comment (hezanathos) [inline README.md:127]: is_valid empty ? Is it possible ? To me it should be either true or false.

Comment (fenrisl) [reply README.md:127]: `is_valid` only takes the value `true` or `false` after the first attempt to connect with the agentless connection and not during the creation of the agentless connection

**Pull request merged**